core/arch/aarch64/codec.txt

```diff
@@ -55,6 +55,7 @@
 
 --------------------------------  impx30     # implicit X30 operand
 --------------------------------  lsl        # implicit LSL for ADD/MOV (immediate)
+--------------------------------  fsz16      # element width of FP vector reg (half (0x2) encoding)
```

Comment:
Is the comment still correct?

Reply:
Updated the comment to make it clear this is mostly used to distinguish between FP16 and single and double encodings in a symmetric fashion